src/js/views/globals/root_views.js

```diff
@@ -61,22 +65,22 @@ const TopRegionView = View.extend({
 
 const PreloaderRegionView = TopRegionView.extend({
   // NOTE: ensures preload can't close on user click
-  contains() {
-    return this.getRegion('region').hasView();
-  },
+  contains: preventRegionClose,
   onRender() {
     this.showChildView('region', new PreloaderView());
   },
 });
 
 const ModalRegionView = TopRegionView.extend({
+  className: 'fill-window--dark',
   behaviors: [{
     behaviorClass: TopRegionBehavior,
-    className: 'fill-window--dark',
+    className: 'is-shown',
```

Comment:
These changes causing the modal test oddity?

Now every modal has the `.fill-window--dark` class name, even if it's not being shown.  Before it only had that class if it was being shown. So Cypress is `get()`ing more elements now than it was before.